src/prefix.js

```diff
@@ -10,11 +10,11 @@ let css = ''
 
 // We should not do anything if required serverside.
 if (isInBrowser) {
-  // Order matters. We need to check Webkit the last one because
-  // other vendors use to add Webkit prefixes to some properties
+  /** Order matters. We need to check Webkit the last one because
```

Comment:
use single line comments inside of blocks - allows easy multiline commenting/decommenting while debugging.

Reply:
multiline  - only top level functions